src/com/google/JsComp.gwt.xml

```diff
@@ -0,0 +1,12 @@
+<module rename-to="jscomp">
+  <inherits name="com.google.gwt.user.User" />
+  <inherits name="com.google.common.base.Base"/>
+  <inherits name="com.google.common.collect.Collect"/>
+  <source path="debugging/sourcemap" />
+  <source path="javascript/jscomp" excludes=".svn,.git,**/ant/**,**/refactoring/**,**/webservice/**,**/testing/**" />
+  <source path="javascript/rhino" excludes=".svn,.git,**/testing/**" />
+  <super-source path="javascript/jscomp/gwt/super" />
```

Comment:
Maybe better wait for J2Cl than invest time on gwt.xml generation 😉

With multiple modules, it would probably be "cleaner" to split super sources in separate folders, but not necessary.